drivers/hmis_external_apis/lib/tasks/export.rake

```diff
@@ -11,5 +11,6 @@ namespace :export do
     HmisExternalApis::AcHmis::DataWarehouseUploadJob.perform_later('project_crosswalk')
     HmisExternalApis::AcHmis::DataWarehouseUploadJob.perform_later('move_in_addresses')
     HmisExternalApis::AcHmis::DataWarehouseUploadJob.perform_later('postings')
+    HmisExternalApis::AcHmis::DataWarehouseUploadJob.perform_later('pathways')
```

Comment:
This is fine, but it's starting to feel like:
```
[
  'clients_with_mci_ids_and_address',
  'hmis_csv_export',
  'project_crosswalk',
  'move_in_addresses',
  'postings',
  'pathways',
].each do |export|
  HmisExternalApis::AcHmis::DataWarehouseUploadJob.perform_later(export)
end
```
might be worth doing.

Reply:
It was a nice idea but we are doing them all at the same time. I'm just going to make the argument optional so if its not passed, it does all of them. sound ok?

Reply:
or maybe there was some reason Todd did it this way so that they would run in separate Delayed Jobs, and be queued separately? It seems fine to me to do them all at once.

Reply:
calling `.perform_later` just queues them up in delayed job, so as written it will generate a job for each one. Looping over the array will not change the functionality, only cleanup the code duplication.

Reply:
I meant if I combined them so **all** the exports would run with only one call to `perform_later`. I'll just leave it as you suggested though.